### PR TITLE
Cleaner recaptcha v2

### PR DESCRIPTION
**Pull request by AmazingDreams**
Cherry picked commits from #2672

### CODE REVIEW

**Comment by AmazingDreams** [comment]
Oh now composer deps are whining..

**Comment by AmazingDreams** [comment]
@TheSerapher I guess it's ready. Let me know what you think.

File: include/pages/login.inc.php [hunk 1]
```diff
@@ -1,20 +1,25 @@
 <?php
 $defflip = (!cfip()) ? exit(header('HTTP/1.1 401 Unauthorized')) : 1;
 
+$recaptcha_enabled = ($setting->getValue('recaptcha_enabled') && $setting->getValue('recaptcha_enabled_logins'));
```
Comment (TheSerapher) [inline include/pages/login.inc.php]: Good idea, never had this idea when writing MPOS.

**Comment by TheSerapher** [comment]
Excellent work man! I have no reason not to merge so ... merged! Lets see if people can test it against development!

File: include/pages/login.inc.php (hunk 1 above)
Comment (AmazingDreams) [reply include/pages/login.inc.php]: The perks of being lazy ;)

**Comment by AmazingDreams** [comment]
Thanks for merging. I'm running this in production on https://pool.amazingsystems.nl/

**Pull request merged**